### PR TITLE
Update accessibility metadata parsing

### PR DESCRIPTION
**Pull request by qnga**

### CODE REVIEW

**Review by mickael-menu — CHANGES_REQUESTED** [review]
Looking good. I would also add the profile mappings as handled there:

https://github.com/readium/kotlin-toolkit/blob/f9a302aa1ad3037eeab33626e65fc7cd61bf4afb/readium/streamer/src/main/java/org/readium/r2/streamer/parser/epub/AccessibilityAdapter.kt#L145-L167

File: streamer/parser/a11y-metadata-parsing.md [hunk 1]
```diff
@@ -74,91 +81,111 @@ When it is a multiple:
...
-These strings must be valid URLs. 
+Values are likely to point to some version of the EPUB accessibility specification and some WCAG profile.
+Try to canonicalize the values to well-known URLs.
 
-Although this URL can be arbitrary, it is likely one of the following:
+Replace any of the following strings with "http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a":
+- "EPUB Accessibility 1.1 - WCAG 2.0 Level A"
+- "http://idpf.org/epub/a11y/accessibility-20170105.html#wcag-a"
+- "http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-a"
+- "http://wwwidpf.org/epub/a11y/accessibility-20170105.html#wcag-a"
```
Comment (mickael-menu) [inline streamer/parser/a11y-metadata-parsing.md:152]: I guess this instead?

```suggestion
- "https://idpf.org/epub/a11y/accessibility-20170105.html#wcag-a"
```

**Review by mickael-menu — CHANGES_REQUESTED** [review]

File: streamer/parser/a11y-metadata-parsing.md (hunk 1 above)
Comment (qnga) [reply streamer/parser/a11y-metadata-parsing.md]: Thanks. Stupid copy-paste from old code already fixed.

**Review by mickael-menu — APPROVED** [review]

**Comment by mickael-menu** [comment]
I don't expect you will get more reviews, so let's merge it and open a new PR if there are more changes to be made.

**Pull request merged**